src/include/function/uuid/functions/gen_random_uuid.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "common/types/uuid.h"
+#include "main/client_context.h"
+
+namespace kuzu {
+namespace main {
+class ClientContext;
+}
+
+namespace common {
+struct UUID;
+struct uuid_t;
+} // namespace common
+
+namespace function {
+
+struct GenRandomUUID {
+    static inline void operation(common::ValueVector& result, void* dataPtr) {
+        KU_ASSERT(result.state->isFlat());
+        auto resultValues = (common::uuid_t*)result.getData();
```

Comment:
Double check with @acquamarin, if UUID is use STRING layout, we should add through `StringVector::addString` interface to handle overflow properly.